src/shared/views/plugins/DeepZoom/osd.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
```

Comment:
Nice work. All working smoothly. Lets take a look at this in more detail when I am back.

Ace work getting it work. It rocks!